newa/cli.py

```diff
@@ -166,13 +204,20 @@ def cmd_event(ctx: CLIContext, errata_ids: list[str], compose_ids: list[str]) ->
             event = Event(type_=EventType.ERRATUM, id=erratum_id)
             errata = ErrataTool(url=et_url).get_errata(event)
             for erratum in errata:
-                # identify compose to be used, just a dump conversion for now
-                compose = erratum.release.strip()
-                if compose.endswith('.GA'):
-                    compose = compose[:-3]
-                compose += '-Nightly'
-                # handle compose differences between ET and TF
-                compose = compose.replace('RHEL-10.0.BETA', 'RHEL-10-Beta')
+                release = erratum.release.strip()
+                # when compose_mapping is provided, apply it
+                if compose_mapping:
+                    compose = apply_mapping(release, compose_mapping, regexp=False)
+                else:
```

Comment:
Isn't this branch unnecessary? `apply_mapping` takes care of the empty compose_mapping itself.

Reply:
The major difference here is different `regexp` value. I could remove the list, true.

Reply:
Updated